docs/cli/creat-score-card.md

```diff
@@ -0,0 +1,131 @@
+# rai insight setup instruction
```

Comment:
Can we have an 'e' on the end of 'create' in the filename, please?

Reply:
fixed!